hbase-client/src/main/java/org/apache/hadoop/hbase/client/BufferedMutatorParams.java

```diff
@@ -70,6 +70,15 @@ public int getRpcTimeout() {
     return rpcTimeout;
   }
 
+  public BufferedMutatorParams operationTimeout(final int operationTimeout) {
+    this.operationTimeout = operationTimeout;
+    return this;
+  }
+
+  /**
+   * @deprecated Since 3.0.0, will be removed in 4.0.0. Use {@link #operationTimeout()}
```

Comment:
Let's do this on branch-2 also so this should be 2.3.0.

Reply:
Fixed to include `2.3.0`.